src/_extensions/acceptance-criteria.md

```diff
@@ -9,20 +9,38 @@ license: cc-by-3.0
 ## Contents
 
 - [Introduction](#intro)
+- [One goal policy](#one-goal-policy)
 - [Extension Metadata](#metadata)
 - [Content](#content)
 - [Development](#development)
 - [Monetization and data processing](#monetization)
 
 ## Introduction {#intro}
 
-Your extension must satisfy all the following acceptance criteria to be included in Opera Addons.
+Your extension must satisfy all the following acceptance criteria to be included in Opera Addons. When these criteria get more specific or restricted, authors of currently published extensions are required to upload updated packages.
+
+All researchers and Opera users are invited to report improper extensions by using the "Reported issues" functionality on respective addon pages.
+
+## One goal policy {#one-goal-policy}
+
+Your extension needs to have one, clearly stated purpose. Proper examples include:
+- display a weather forecast after clicking a button
+- show a popup if an item can be purchased in a better price somewhere else
+- assist in downloading files from chosen websites
+
+Examples of prohibited behaviours:
+- additional bundling of "shopping assistants"
+- a sidebar extension with a calculator includes a website blocker
+
+The [browser action button](https://developer.chrome.com/extensions/browserAction) should only serve as a shortcut to the extension’s main functionality. For instance, if an extension's single purpose is downloading video, clicking the browser action icon should launch download process.
```

Comment:
This line mixes two variants of ' in one line. Not sure which is the most common otherwise. There is one more "curly" further down in Opera's default startpage.